package.json

```diff
@@ -2,28 +2,25 @@
   "name": "spectacle",
   "version": "5.5.0",
   "description": "ReactJS Powered Presentation Framework",
-  "main": "lib/index.js",
-  "module": "es/index.js",
+  "source": "src/index.js",
+  "main": "dist/umd/spectacle.js",
   "types": "index.d.ts",
   "scripts": {
-    "clean": "rimraf dist lib es",
+    "clean": "rimraf dist",
```

Comment:
We shouldn't remove `lib es` from clean until our CJS and ESM replacement PRs land.